jetty-io/src/test/java/org/eclipse/jetty/io/SelectorManagerTest.java

```diff
@@ -117,8 +121,7 @@ protected void connectionFailed(SelectableChannel channel, Throwable ex, Object
             SocketChannel client1 = SocketChannel.open();
             client1.configureBlocking(false);
             client1.connect(address);
-            long timeout = connectTimeout * 2;
-            timeoutConnection.set(timeout);
+            timeoutConnection.set(connectTimeout * 110 / 100);
```

Comment:
Again, 100 ms difference is too small.
You would need to wait until `latch1` is counted down, then release the thread in `doFinishConnect()`.
Also, `3 * idleTimeout` should probably be changed (see previous comment).

Reply:
It's the same problem as above: it doesn't matter if it's 100ms more or 1s more as long as the connection delay is higher than the timeout.